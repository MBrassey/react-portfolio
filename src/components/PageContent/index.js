import ParticlesBg from "particles-bg";
import React from "react";

const PageContent = (props) => {
  return (
    <div>
      <ParticlesBg type="cobweb" bg={true} color="#2d93ca" />
      {props.children}
    </div>
  );
};

export default PageContent;
