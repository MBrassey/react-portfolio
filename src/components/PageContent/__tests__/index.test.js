import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import React from 'react';

import PageContent from '../../PageContent';

afterEach(cleanup);

describe('Page component', () => {
  it('renders', () => {
    render(<PageContent />);
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<PageContent />);

    expect(asFragment()).toMatchSnapshot();
  });
})
