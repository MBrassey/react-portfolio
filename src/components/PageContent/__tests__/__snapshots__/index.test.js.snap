// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page component matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <canvas
      style="width: 100%; height: 100%; position: absolute; z-index: -1; top: 0px; left: 0px;"
    />
  </div>
</DocumentFragment>
`;
